client/views/settings/website-management/partials/tenant.html

```diff
@@ -1,6 +1,6 @@
 <div class="sd-grid-item-header">
     <h3 class="sd-grid-item-header__heading sd-overflow-ellipsis">
-        <a ng-href="{{site.outputChannel ? site.outputChannel.config.url : site.subdomain ? 'http://' + site.subdomain + '.' + site.domainName: 'http://' + site.domainName}}" flow="down">
+        <a target="_blank" ng-href="{{site.outputChannel ? site.outputChannel.config.url : site.subdomain ? 'http://' + site.subdomain + '.' + site.domainName: 'http://' + site.domainName}}" flow="down">
```

Comment:
protocol should be omitted

Reply:
it is an url

Reply:
yep but still it should open website according to the current protocol

Reply:
nope bcs website can handle http only where superdesk instance should use https by default.